addon/services/tour.js

```diff
@@ -449,16 +449,19 @@ export default Service.extend(Evented, {
       const currentStep = tour.steps[index];
 
       currentStep.on('before-show', () => {
-        if (this.get('modal')) {
-          const currentElement = this.getElementForStep(currentStep);
-
-          if (currentElement) {
+        const currentElement = this.getElementForStep(currentStep);
+        if (currentElement) {
+          if (this.get('modal')) {
```

Comment:
I pushed an update that reorganized these checks into the body of `popoutElement`. A consequence is that the condition for `createHighlightOverlay`, like the condition for `highlightClass`, is now independent of whether the tour is `modal == true` or not. Let me know if you foresee this change being a problem of its own